src/components/DropdownButton/DropdownButton.module.css

```diff
@@ -0,0 +1,45 @@
+@import '../../design-tokens/mixins.css';
+
+
+/*------------------------------------*\
+    #DROPDOWN BUTTON
+\*------------------------------------*/
+
+/**
+ * The button to trigger the display of the dropdown.
+ * 1) Place button text on the left and the expand more icon on the right.
+ */
+.dropdown-button {
+  width: 100%;
+  padding: var(--eds-size-1);
+
+  border: var(--eds-theme-border-width) solid var(--eds-theme-color-border-neutral-default);
+  border-radius: var(--eds-theme-border-radius);
+
+  display: flex;
+  justify-content: space-between; /* 1 */
+  align-items: center;
+
+  cursor: pointer;
+
+  background-color: var(--eds-theme-color-background-neutral-default);
+
+  @mixin eds-theme-typography-body-text-md;
+}
+
+/**
+ * The caret icon to decorate the dropdown button, animated to rotate.
+ * 1) Turns off animation if user prefers reduced motion
+ */
+.dropdown-button__icon {
+  transform: rotate(0);
+
+  transition: transform var(--eds-anim-move-quick) ease-out;
+  @media (prefers-reduced-motion) {
+    transition-property: none; /* 1 */
```

Comment:
Just curious – why did you decide to use `transition-property: none` (as opposed to `transition: none` or `transition-duration: 0`)? I don't think any specific way is superior, but I do kinda want us to pick one and stick with it. (I did hear an argument to use `transition-duration: 0.0001` or something which is that if there's some code that is checking for when the transition ends, reducing the duration will still trigger that code whereas removing the transition altogether won't. But we don't usually have logic like that, so I dunno if I really want to optimize for it.)

Reply:
Good question, the `transition-none` TW utility class is `transition-property: none`, so I just used that, but if `transition: none` also disables transition animations, I will go with that since it feels more self explanatory imo